plugins/callback/foreman.py

```diff
@@ -249,8 +262,11 @@ def send_reports(self, stats):
                     },
                     "logs": list(build_log(self.items[host])),
                     "reporter": "ansible",
+                    "check_mode": self.check_mode,
                 }
             }
+            if self.check_mode == '1':
+                report['config_report']['status']['pending'] = total['ok']
```

Comment:
should we also se ok to 0? otherwise we'd double count tasks or not?

Reply:
Yes, I think you are right, I will add it.

Reply:
I am not sure this is necessary because there is no other use of the 'ok' tasks. the status focuses on the changed, failed, and skipped tasks only.